fbw-common/src/systems/instruments/src/EFB/Dashboard/Widgets/WeatherWidget.tsx

```diff
@@ -4,316 +4,307 @@
-    name: 'origin'|'destination';
-    simbriefIcao: string;
-    userIcao?: string
+  name: 'origin' | 'destination';
+  simbriefIcao: string;
+  userIcao?: string;
 }
 
 export const WeatherWidget: FC<WeatherWidgetProps> = ({ name, simbriefIcao, userIcao }) => {
-    const [baroType] = usePersistentProperty('CONFIG_INIT_BARO_UNIT', 'HPA');
-    const dispatch = useAppDispatch();
-    const [simbriefIcaoAtLoading, setSimbriefIcaoAtLoading] = useState(simbriefIcao);
-    const [metarSource] = usePersistentProperty('CONFIG_METAR_SRC', 'MSFS');
-    const [metarError, setErrorMetar] = useState('');
-    const [usingColoredMetar] = usePersistentNumberProperty('EFB_USING_COLOREDMETAR', 1);
-    const source = metarSource;
+  const [baroType] = usePersistentProperty('CONFIG_INIT_BARO_UNIT', 'HPA');
+  const dispatch = useAppDispatch();
+  const [simbriefIcaoAtLoading, setSimbriefIcaoAtLoading] = useState(simbriefIcao);
+  const [metarSource] = usePersistentProperty('CONFIG_METAR_SRC', 'MSFS');
+  const [metarError, setErrorMetar] = useState('');
+  const [usingColoredMetar] = usePersistentNumberProperty('EFB_USING_COLOREDMETAR', 1);
+  const source = metarSource;
 
-    const getBaroTypeForAirport = (icao: string) => (['K', 'C', 'M', 'P', 'RJ', 'RO', 'TI', 'TJ']
-        .some((r) => icao.toUpperCase().startsWith(r)) ? 'IN HG' : 'HPA');
+  const getBaroTypeForAirport = (icao: string) =>
+    ['K', 'C', 'M', 'P', 'RJ', 'RO', 'TI', 'TJ'].some((r) => icao.toUpperCase().startsWith(r)) ? 'IN HG' : 'HPA';
 
-    const metar = useAppSelector((state) => (name === 'origin' ? state.dashboard.departureMetar : state.dashboard.destinationMetar)) ?? MetarParserTypeProp;
-    const setMetar = name === 'origin' ? setDepartureMetar : setDestinationMetar;
+  const metar =
+    useAppSelector((state) =>
+      name === 'origin' ? state.dashboard.departureMetar : state.dashboard.destinationMetar,
+    ) ?? MetarParserTypeProp;
+  const setMetar = name === 'origin' ? setDepartureMetar : setDestinationMetar;
 
-    const [showMetar, setShowMetar] = usePersistentNumberProperty(`CONFIG_SHOW_METAR_${name}`, 0);
+  const [showMetar, setShowMetar] = usePersistentNumberProperty(`CONFIG_SHOW_METAR_${name}`, 0);
 
-    const BaroValue = () => {
-        const displayedBaroType = baroType === 'AUTO' ? getBaroTypeForAirport(metar.icao) : baroType;
-        if (displayedBaroType === 'IN HG') {
-            return (
-                <>
-                    {metar.barometer.hg.toFixed(2)}
-                    {' '}
-                    inHg
-                </>
-            );
-        }
-        return (
-            <>
-                {metar.barometer.mb.toFixed(0)}
-                {' '}
-                mb
-            </>
-        );
-    };
+  const BaroValue = () => {
+    const displayedBaroType = baroType === 'AUTO' ? getBaroTypeForAirport(metar.icao) : baroType;
+    if (displayedBaroType === 'IN HG') {
+      return <>{metar.barometer.hg.toFixed(2)} inHg</>;
```

Comment:
While this is more concise it is also easier to exclude the space in future changes, just note this.